docs/localization/LOCALIZATION.md

```diff
@@ -1,52 +1,58 @@
+# Otter Localization
 
-# About
-
-Localization module is built on top of an open source [ngx-translate](https://github.com/ngx-translate/core) library.
+The localization module is built on top of an open source [ngx-translate](https://github.com/ngx-translate/core) library.
 
 [Date](https://angular.io/api/common/DatePipe), [currency](https://angular.io/api/common/CurrencyPipe), [decimal](https://angular.io/api/common/DecimalPipe) number, [pluralization](https://angular.io/api/common/I18nPluralPipe) and [selection](https://angular.io/api/common/I18nSelectPipe) can be easily handled by built-in Angular pipes and by providing them a locale.
 
 # Features
 
 - Multiple [locales](https://github.com/angular/angular/tree/master/packages/common/locales) support, switchable at runtime
 
-- RTL text direction support
-
-- Translations loader using specific endpoint (external URL source like CMS for example) or defaulting to local translation bundles in **src/assets/assets-otter/i18n** folder of your application. Bundles input folder can be changed by configuration (via shell option).
+- RTL (right-to-left) text direction support
 
-- Fallback language. In case some resource key does not exist in language X, the loader tries to fetch translation bundle from the endpoint location in fallback language Y in the first place and ends by loading json bundles from the application (root of dist folder **/** by default) if previous steps have failed (bundles output path in dist folder is also configurable via a shell option).
+- Translations loader using a specific endpoint (external URL source such as a CMS for example) or defaulting to local translation bundles in the **src/assets/assets-otter/i18n** folder of your application. The input folder for bundles can be changed by configuration (via a shell option).
```

Comment:
which shell option ? (not sure to know what it is)

Reply:
I changed to CLI option to be more clear (`The assets folder for bundles can be changed by configuration (via a CLI option).`), but I could remove this line if you don't find it necessary.